.github/workflows/lint.yaml

```diff
@@ -58,7 +58,7 @@ jobs:
     steps:
       - id: setup
         name: Setup
-        uses: scientist-softserv/actions/setup-env@v0.0.14
+        uses: scientist-softserv/actions/setup-env@testing-deploy-env-vars
```

Comment:
This is pointing to our testing branch, once the main PR is approved we will need to cut this into a version.